django_socio_grpc/protobuf/generators.py

```diff
@@ -31,21 +38,18 @@ def print(self, message, verbose_level=0):
 
     def get_protos_by_app(self):
         proto_by_app = {}
-        for app_name, registered_items in self.registry_instance.registered_app.items():
+        for app_name, registry in self.registry_instance.registered_app.items():
```

Comment:
why renaming registered_items ? is registered_app a dict of registry instance now ?

Reply:
It is a dict[str, AppHandlerRegistry]